yunwoo/programmers-1/x만큼-간격이-있는-n개의-숫자.js

```diff
@@ -0,0 +1,12 @@
+function solution(x, n) {
+  var result = [];
+  for (let i = 1; i <= n; i++) {
+    result.push(x * i);
+  }
+  return result;
+}
```

Comment:
```suggestion
function solution(x, n) {
  const result = [];
  
  for (let i = 1; i <= n; i++) {
    result.push(x * i);
  }
  
  return result;
}
```
- 잘푸셨습니다 👍 다만 아쉬운건 result 변수가 `var`로 선언이 됐다는 점이 아쉽네요 🥲 이미 var가 아닌 let과 const를 사용해야되는거 잘 알고 계셔서 아마 실수를 하신 것 같은데 이런 실수를 방지하고 좋은 습관을 들이기위해 제출 전에 3번 검토 꼭 해주세요!!~🙇🏻‍♂️

Reply:
- 헉 이런 실수를.. 죄송합니다 검토한다고 했는데 변수명에만 신경썼던거 같습니다🥲 다음에 더 꼼꼼히 확인하겠습니다 3번 검토!!!
- 문단별 분리도 안되어있네요 반성하겠습니다😂
- 오.. 첫번째 인자가 배열의 개수, 2번째는 콜백함수, 콜백함수 인자의 첫번째 _ <- 요거는 무엇인지 궁금합니다.. value 라면 _요소로 표시하신 이유가 있는지도 궁금합니다!!

Reply:
- 와 그런뜻이 있었군요.. 올려주신 참고링크도 잘봤습니다 감사합니다🙇‍♂️🙇‍♂️